libpod/container_api.go

```diff
@@ -847,31 +843,57 @@ func (c *Container) ShouldRestart(ctx context.Context) bool {
 	return c.shouldRestart()
 }
 
-// ResolvePath resolves the specified path on the root for the container.  The
-// root must either be the mounted image of the container or the already
-// mounted container storage.
-//
-// It returns the resolved root and the resolved path.  Note that the path may
-// resolve to the container's mount point or to a volume or bind mount.
-func (c *Container) ResolvePath(ctx context.Context, root string, path string) (string, string, error) {
-	logrus.Debugf("Resolving path %q (root %q) on container %s", path, root, c.ID())
+// CopyFromArchive copies the contents of the specified tarStream to path.
+func (c *Container) CopyFromArchive(ctx context.Context, path string, tarStream io.Reader) (func() error, error) {
```

Comment:
Can you make it more clear that this one copies into the container, the other one copies out of it? Updating the comments on both could help.